Client/javascripts/es6/show.js

```diff
@@ -0,0 +1,71 @@
+const baseUrl = 'http://localhost:3000/api/v1';
+document.addEventListener('DOMContentLoaded', () => {
+	const pageUrl = window.location.href;
+	const url = new URL(pageUrl);
+	const entryId = url.searchParams.get('entries');
+	fetch(`${baseUrl}/entries/${entryId}`, {
+		method: 'GET',
+		headers: {
+			'Content-Type': 'application/json',
+			token: localStorage.getItem('diary_token'),
+		},
+	})
+		.then(res => res.json())
+		.then((data) => {
+			console.log(data);
+			document.querySelector('body').insertAdjacentHTML('afterbegin', '<img src="images/Rolling.svg" id="loading" />');
+			if (data.status === 'Failed') {
+				document.getElementById('loading').style.display = 'none';
+				document.getElementById('flash-message').style.display = 'block';
```

Comment:
You can define your selectors as a global variable and then reuse them wherever you want to

Reply:
Okay, will do so